tst/teststandard/bugfix.tst

```diff
@@ -1826,6 +1826,10 @@ true
 gap> Log2Int( 2^60 );
 60
 
+# Reported by Chris Jefferson on 20151008 in github issue #282
```

Comment:
I suggest to follow the date format used in other entries in this file consistently - `2015/10/08` is easier to parse for humans (I don't mind switching to ISO either).